packages/react/src/utils/componentUtilityFunctions.js

```diff
@@ -554,3 +555,27 @@ export const getMultiSelectItems = (column, filterValue) => {
 
   return [];
 };
+
+/**
+ * Hook to detect clicks outside of a specified element
+ * @param {node} ref Upon interaction with this DOM node handler won't be invoked
+ * @param {function} handler Callback for interaction event
+ */
+export const useOnClickOutside = (ref, handler) => {
+  useEffect(() => {
+    const listener = (event) => {
+      if (!ref.current || ref.current.contains(event.target)) {
+        return;
+      }
+      handler(event);
+    };
+
+    document.addEventListener('mousedown', listener);
+    document.addEventListener('touchstart', listener);
+
+    return () => {
+      document.removeEventListener('mousedown', listener);
+      document.removeEventListener('touchstart', listener);
+    };
+  }, [ref, handler]);
```

Comment:
Would ref ever be a different instance (vs ref.current?)

Reply:
@sls-ca Thanks for commenting on this! It was a bug.